Example/TABTestKit.xcodeproj/xcshareddata/xcschemes/TABTestKit_ExampleUITests.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```

Comment:
what is this?

Reply:
I think this gets added with Swift 4 conversion? It's weird but it needs to be there for some reason